SimPEG/DCIP/DCIPUtils.py

```diff
@@ -179,7 +179,7 @@ def plot_pseudoSection(DCsurvey, axs, stype):
         Input:
         :param d2D, z0
         :switch stype -> Either 'pdp' (pole-dipole) | 'dpdp' (dipole-dipole)
```

Comment:
These flags are not very self-explanatory, I would suggest
`surveyType = 'pole-dipole' or 'dipole-dipole'`
`dataType = `appConductivity`, `appResistivity` or `potential`

Thoughts?